.gitignore

```diff
@@ -1 +1 @@
-/node_modules/
+node_modules
```

Comment:
If you add a `/` to the end of this, it will ignore everything in the node_modules directory. So make it `node_modules/`